src/react/components/pages/editorPage/editorPage.test.tsx

```diff
@@ -510,15 +510,29 @@ describe("Editor Page Component", () => {
             const wrapper = createComponent(store, MockFactory.editorPageProps());
             await waitForSelectedAsset(wrapper);
 
-            wrapper.update();
-
-            const editorPage = wrapper.find(EditorPage).childAt(0);
+            expect(editorPage.state().selectedTag).toBeNull();
 
             dispatchKeyEvent("1");
 
-            expect(editorPage.state().lockedTags).toEqual([]);
             expect(editorPage.state().selectedTag).toEqual(project.tags[0].name);
+        });
+
+        it("does not set selected tag or locked tags when hot key is pressed", async () => {
```

Comment:
test not related to log tags, we should either update test or update description